Tools/PRWatcher.ps1

```diff
@@ -0,0 +1,190 @@
+#Streamlines Winget-pkgs manifest PR moderator approval by watching the clipboard - copy a PR title to your clipboard, and Watch-PRTitles attempts to parse the PackageIdentifier and version number, gathers the version from Winget, and gives feedback in your Powershell console. Also outputs valid titles to a logging file. Freeing moderators to focus on approving and helping. 
+
+$MainFolder = "C:\ManVal" #Created in the manual validation pipeline
```

Comment:
Script throws an error if this isn't created first. Might be good to have a check that ensures the folder is created, or to create if non-existing

Reply:
This has been fixed.